### PR TITLE
Sensitivity analysis

### PR DESCRIPTION
**Pull request by ch1eroe1**
Added sensitivity analysis to 07-analysis.php and modified the pecan.xml file to add the sensitivity settings to the configuration (04-runpecan.php)
#517 is the git pecan issue that this branch is associated with.

### CODE REVIEW

File: web/04-runpecan.php [hunk 1]
```diff
@@ -68,7 +68,14 @@
 if (!isset($_REQUEST['ensemble'])) {
   die("Need an ensemble value.");
 }
-$ensemble=$_REQUEST['ensemble'];
+$ensemble = $_REQUEST['ensemble'];
+
+$sensitivity = array();
+if (!isset($_REQUEST['sensitivity'])) {
+  die("Need a sensitivity value-comma seperated.");
```
Comment (dlebauer) [inline web/04-runpecan.php]: Not clear from message what type of value is expected required here... are these quantiles in a comma separated list? You could say something like '1, 5, 33, 66, 95, 99 is the default'

Comment (mdietze) [inline web/04-runpecan.php]: I agree that this is unclear but @dlebauer I don't think your example actually makes this more clear. Also, I don't think these are good defaults (default should be that the SA is turned off). Looking at the code below it seems that what's expected is actually sigma equivalents, which is OK but we do need the ability to enter quantiles as well.  This design is an OK first pass (and I'm happy to pull it once there's more explanation about what's expected), but ultimately entering a comma-separated list doesn't seem like the best design. On one hand, it's going to get really annoying to have to type in if you're doing a lot of this. On the other hand, something more graphical might provide a more intuitive method for specifying quantile/sigma points.

More generally, I didn't realize until I pulled the latest code that the Ensembles and SA are being entered on a completely new page that was a default behavior. I think the default behavior should be to skip this menu entirely, and only go here if you select some sort of "Additional Settings" box or something like that.  Right now we're getting a lot of stuff creeping into the UI that really doesn't have to be there for most runs and that's making the overall design less clean. We're really overdue to sit down and discuss what we want the UI to look like, how we want it to function, and how we want to accommodate features we know are coming down the pipeline.

File: web/04-runpecan.php [hunk 2]
```diff
@@ -222,6 +229,16 @@
 fwrite($fh, "    <variable>NPP</variable>" . PHP_EOL);
 fwrite($fh, "  </ensemble>" . PHP_EOL);
```
Comment (robkooper) [inline web/04-runpecan.php]: if sensitivity == "" do not add sensitivity.analysis

File: web/07-analysis.php [hunk 3]
```diff
@@ -190,6 +190,9 @@ function mapsLoaded() {
       <label>Ensemble<sup>*</sup></label>
       <input type="text" name="ensemble" id="ensemble" value="<?php echo 1; ?>" onChange="validate();"/>
       <div class="spacer"></div>
+      <label>Sensitivity<sup>*</sup></label>
+      <input type="text" name="sensitivity" id="sensitivity" value="<?php echo "-1,1"; ?>" onChange="validate();"/>
```
Comment (robkooper) [inline web/07-analysis.php]: add title with some text, for example
"Allows to set the sensitivity analysis values as sigma, for example -1, 0, 1"

**Pull request merged**